src/content/docs/security/security-privacy/compliance/certificates-standards-regulations/fedramp-moderate.mdx

```diff
@@ -207,6 +207,19 @@ The following services are not FedRAMP-authorized:
       <td>
         [New Relic Mobile App](/docs/mobile-apps/mobile-apps-intro/#mobile-app-features)
       </td>
+
+    <tr>
+      <td>
+        N/A
+      </td>
+
+      <td>
+        AWS
+      </td>
+
+      <td>
+        [Historical Data Export](/docs/apis/nerdgraph/examples/nerdgraph-historical-data-export)
+      </td>
```

Comment:
And this is the main change.

Reply:
@rbayersdorfer-newrelic thanks for adding this! I had to add a </tr> to get this to build, but otherwise, it looks great!